### PR TITLE
mimic: tests: do not take ceph.conf.template from ceph/teuthology.git

### PR DESCRIPTION
**Pull request by smithfarm**
backport tracker: https://tracker.ceph.com/issues/42263

---

parent tracker: https://tracker.ceph.com/issues/42261

this backport was staged using ceph-backport.sh version 15.0.0.5775
find the latest version at https://github.com/ceph/ceph/blob/master/src/script/ceph-backport.sh

### CODE REVIEW

**Comment by smithfarm** [comment]
@neha-ojha Ack. I think it's fixed now - please take another look?

**Comment by smithfarm** [comment]
@neha-ojha Ping, my question above

**Comment by yuriw** [comment]
[wip-yuri4-testing-2019-10-21-1527-mimic](https://trello.com/c/fqlT4ZTu/727-wip-yuri4-testing-2019-10-21-1527-mimic)

**Comment by smithfarm** [comment]
Note: 0483c1c was omitted from https://github.com/ceph/ceph/pull/30180 so I added it here.

File: qa/tasks/ceph.py [hunk 1]
```diff
@@ -381,6 +383,114 @@ def cephfs_setup(ctx, config):
...
+    args = [
+        'adjust-ulimits',
+        'ceph-coverage',
+        '{tdir}/archive/coverage'.format(tdir=testdir),
+        'monmaptool',
+        '--create',
+        '--clobber',
+    ]
+    for (name, addr) in addresses:
+        n = name[4:]
```
Comment (dillaman) [inline qa/tasks/ceph.py]: I think the name could be `cluster1.mon.a` so fixing this to 4 would be incorrect. Need `teuthology.split_role(name)`?

**Comment by smithfarm** [comment]
Thanks for the hints, @dillaman - see https://github.com/ceph/ceph/pull/31424

**Pull request merged**